docs/guide/bundles.md

```diff
@@ -24,59 +24,106 @@ To learn more about organizing your eZ Platform project, see [Best Practices](be
-
-- `ezsystems/platform-ui-bundle` for most management interfaces: Sections, Content Types, Roles, Policies, etc.
-- `ezsystems/ezpublish-kernel` for user registration and user generated content
-
-!!! enterprise
-
-    #### StudioUIBundle
-
-    StudioUIBundle contains the Studio editing interface
-    provided in eZ Platform Enterprise Edition.
-
-    #### Landing Page Field Type Bundle
-
-    EzLandingPageFieldTypeBundle provides
-    the Landing Page that is at the heart of StudioUI.
+The eZ Platform directory contains multiple sub-directories.
+Each sub-directory is dedicated to a specific part of the system and contains a collection of logically related files.
 
 !!! tip
 
     You can see the bundles that are automatically installed with eZ Platform in [composer.json](https://github.com/ezsystems/ezplatform/blob/master/composer.json).
+
+The following tables give an overview of the main eZ Platform directories.
+
+### Core bundles
+
+|Bundle|Description|
+|---------|-----------|
+|[ezplatform](https://github.com/ezsystems/ezplatform)|meta repository that pulls in all dependencies for clean distribution of eZ Platform|
+|[ezpublish-kernel](https://github.com/ezsystems/ezpublish-kernel)|contains the core of the whole eZ Platform application e.g. EzPublishCoreBundle|
+|[repository-forms](https://github.com/ezsystems/repository-forms)|provides form-based interaction for the Repository Value objects|
+|[ezplatform-solr-search-engine](https://github.com/ezsystems/ezplatform-solr-search-engine)|[Solr-powered](http://lucene.apache.org/solr/) search handler for eZ Platform|
+|[ez-support-tools](https://github.com/ezsystems/ez-support-tools)|provides functionality for system information|
+|[ezplatform-design-engine](https://github.com/ezsystems/ezplatform-design-engine)|design fallback system for eZ Platform similar to legacy design fallback system|
+|[BehatBundle](https://github.com/ezsystems/BehatBundle)|common reusable sentence implementations and other common needs for Behat testing in eZ bundles/projects|
+
+!!! Enterprise
+
+    |Bundle|Description|
+    |---------|-----------|
+    |[ezplatform-ee](https://github.com/ezsystems/ezplatform-ee)|fork of the "ezplatform" meta repository, contains changes to composer.json that pull in all dependencies from updates.ez.no for eZ Platform Enterprise Edition (a commercial distribution of eZ Platform with additional features)|
+    |[date-based-publisher](https://github.com/ezsystems/date-based-publisher)|provides the date based publishing functionality for the eZ Studio product|
+    |[flex-workflow](https://github.com/ezsystems/flex-workflow)|implementation of a collaboration feature that lets you send content draft to any user for a review or rewriting|
+
+### Platform usage
+
+|Bundle|Description|
+|---------|-----------|
+|[ez-js-rest-client](https://github.com/ezsystems/ez-js-rest-client)|JavaScript REST client for eZ Platform RESTv2 API, mimics PHP API found in eZ Platform|
```

Comment:
what is the purpose here? Some are already bundled with eZ Platform, like this one, others are not, why are they listed here? Are we trying to make a recommendation on bundles not provided out of the box? ;)